Packs/CortexXDR/Scripts/CortexXDRIdentityInformationWidget/CortexXDRIdentityInformationWidget.py

```diff
@@ -0,0 +1,66 @@
+"""Base Script for Cortex XSOAR (aka Demisto)
+
+This is an empty script with some basic structure according
+to the code conventions.
+
+MAKE SURE YOU REVIEW/REPLACE ALL THE COMMENTS MARKED AS "TODO"
+
+Developer Documentation: https://xsoar.pan.dev/docs/welcome
+Code Conventions: https://xsoar.pan.dev/docs/integrations/code-conventions
+Linting: https://xsoar.pan.dev/docs/integrations/linting
+
+"""
+
+import demistomock as demisto
+from CommonServerPython import *
+
+import traceback
+
+''' COMMAND FUNCTION '''
+
+
+def get_identity_info() -> List[Dict]:
+    context = demisto.context()
+    alerts = demisto.get(context, 'PaloAltoNetworksXDR.OriginalAlert')
+    users = demisto.get(context, 'AWS.IAM.Users')
+    if not users:
+        raise DemistoException('AWS users are not in context')
+    access_keys = users[0].get('AccessKeys', [])
```

Comment:
can `AWS.IAM.Users` have a single user and saved as a `dict` instead of a `list`?

Reply:
No, it's a tuple of two elements. The first contains all the users access keys